Display.lua

```diff
@@ -527,10 +527,16 @@ function WarpDeplete:SetForcesTotal(totalCount)
   self.forcesState.pullPercent = totalCount > 0 and self.forcesState.pullCount / totalCount or 0
 
   local currentPercent = totalCount > 0 and self.forcesState.currentCount / totalCount or 0
-  if currentPercent > 1.0 then currentPercent = 1.0 end
+
+  if not WarpDeplete.db.profile.unclampForcesPercent then
```

Comment:
`self.db` should be used instead of `WarpDeplete.db`, that goes for all other instances as well